src/ConfigFactory.Core/Attributes/NumericConfigAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace ConfigFactory.Core.Attributes;
+
+[AttributeUsage(AttributeTargets.Property)]
+public class NumericConfigAttribute : Attribute
+{
+    /// <summary>
+    /// Minimum numeric value
+    /// </summary>
+    public object? Minimum { get; set; }
+    /// <summary>
+    /// Maximum numeric value
+    /// </summary>
+    public object? Maximum { get; set; }
+    /// <summary>
+    /// Increment numeric value
+    /// </summary>
+    public object? Increment { get; set; }
```

Comment:
Why aren't these decimal values?

```suggestion
    /// <summary>
    /// Minimum numeric value
    /// </summary>
    public decimal Minimum { get; set; } = decimal.MinValue;
    
    /// <summary>
    /// Maximum numeric value
    /// </summary>
    public decimal Maximum { get; set; } = decimal.MaxValue;
    
    /// <summary>
    /// Increment numeric value
    /// </summary>
    public decimal Increment { get; set; } = 1.0;
```

Reply:
See conversation below.